content/guides/use-case/jupyter/run-jupyter.md

```diff
@@ -0,0 +1,175 @@
+
+## Prerequisites
+
+You have installed the latest version of [Docker Desktop](../../../get-docker.md).
+
+## Run a JupyterLab container
+
+In a terminal, run the following command to run your JupyterLab container.
+
+```console
+$ docker run --rm -p 8888:8888 jupyter/base-notebook start-notebook.py --NotebookApp.token='my-token'
+```
+
+The following is a breakdown of the command:
+
+- The `docker run` command tells Docker to pull and run an image. In this case,
+the image is the `jupyter/base-notebook` image.
+- The `-p` option maps port `8888` on the host to port `8888` in the container.
+- `--rm` tells Docker to remove the image after it's stopped.
+- `start-notebook.py --NotebookApp.token='my-token'` is the command that the
+  server will run after starting. In this case, it's a script that sets the
+  access token for convenience. For more details about `start-notebook.py`, see
+  [Jupyter Server Options](https://jupyter-docker-stacks.readthedocs.io/en/latest/using/common.html#jupyter-server-options).
+
+To access the container, in a web browser navigate to [localhost:8888/lab?token=my-token](http://localhost:8888/lab?token=my-token).
+
+After accessing the URL, you should see the JupterLab interface. Since you specified `--rm` when running the container, the container and any data you create is deleted when the container stops.
+
+To stop the container, press `ctrl`+`c` in the terminal. If you see a prompt confirming shutdown in the terminal, specify `Y` and press `Enter`.
+
+## Save and access notebooks locally
+
+When you remove a container, all data in that container is deleted. To save
+notebooks or access notebooks on local machine, you can use a
+[bind mount](../../../storage/bind-mounts.md).
+
+### Run a JupterLab container with a bind mount
+
+Open a terminal, and change directory to a directory where you would like to save the notebook's data. Then, run the following command based on your operating system and shell.
+
+{{< tabs >}}
+{{< tab name="Windows (Command Prompt)" >}}
+
+```console
+$ docker run --rm -p 8888:8888 -v "%cd%":/home/jovyan/work jupyter/base-notebook start-notebook.py --NotebookApp.token='my-token'
+```
+{{< /tab >}}
+{{< tab name="Windows (PowerShell)" >}}
+
+```console
+$ docker run --rm -p 8888:8888 -v "$(pwd):/home/jovyan/work" jupyter/base-notebook start-notebook.py --NotebookApp.token='my-token'
+```
+
+{{< /tab >}}
+{{< tab name="Windows (Git Bash)" >}}
+
+```console
+$ docker run --rm -p 8888:8888 -v "/$(pwd):/home/jovyan/work" jupyter/base-notebook start-notebook.py --NotebookApp.token='my-token'
+```
+
+{{< /tab >}}
+{{< tab name="Mac / Linux" >}}
+
+```console
+$ docker run --rm -p 8888:8888 -v "$(pwd):/home/jovyan/work" jupyter/base-notebook start-notebook.py --NotebookApp.token='my-token'
+```
+
+{{< /tab >}}
+{{< /tabs >}}
+
+The `-v` option tells Docker to provide access to your machine's present working directory (PWD) from the container at `/home/jovyan/work`.
```

Comment:
suggestion: maybe we could expand on the significance of `/home/jovyan` here. I didn't get it and I had to google. It seems like something that's unique to the jupyter Docker image, https://github.com/jupyter/docker-stacks/issues/358